src/main/java/seedu/address/logic/commands/HelpCommand.java

```diff
@@ -9,13 +9,30 @@ public class HelpCommand extends Command {
 
     public static final String COMMAND_WORD = "help";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Shows program usage instructions.\n"
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Shows program usage instructions within application.\n"
             + "Example: " + COMMAND_WORD;
 
-    public static final String SHOWING_HELP_MESSAGE = "Opened help window.";
+    public static final String SHOWING_HELP_MESSAGE = "Here are the list of available commands:\n\n"
+            + "Add: add n\\NAME ic\\IC_NUMBER dob\\DATE_OF_BIRTH ad\\ADMISSION_DATE w\\WARD [t\\TAG]...\n"
```

Comment:
Is it better to use the prefix declared in CliSyntax, i.e. PREFIX_NAME, etc. , so that it's easier to make changes?

Reply:
I feel that because this help message is viewed by normal users, they may not understand parameter names like PREFIX_NAME, PREFIX_IC as it is not as intuitive as just NAME. Should I just remove the underscore separating the words then to make it consistent with the intuitive naming?

Reply:
What I meant was instead of using "n\", can use PREFIX_NAME, the NAME can keep it. The underscore should be ok as well.